.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -0,0 +1,31 @@
+name: "\U0001F680 Feature request:"
+description: Submit a proposal/request for a new transformers feature
+labels: [ "feature" ]
```

Comment:
Only seeing the label for 'bug' in 'bug report' appearing - is the label 'feature' expected to show in 'feature request'?

Reply:
I think this comes from the fact that my fork does not have these labels. Let me add them to my fork to check.

Reply:
Indeed, after creating the label it shows up: 
<img width="1292" alt="image" src="https://user-images.githubusercontent.com/30755778/163874592-936fc577-94c1-4e97-9b8b-d8fe64b7a9da.png">